src/components/GameTabs.vue

```diff
@@ -59,9 +59,16 @@
           </div>
 
           <router-link
-            v-if="!allGamesComplete && !game.completedOn && !game.taskData?.taskURL && !game.taskData?.variantURL"
+            v-if="
+              !allGamesComplete &&
+              !game.completedOn &&
+              !game.taskData?.taskURL &&
+              !game.taskData?.variantURL &&
+              game.taskId !== 'core-tasks'
```

Comment:
```suggestion
```

Reply:
I'll go ahead and commit this change and then approve. But feel free to revert if I'm in error.